File: src/main/java/com/refinedmods/refinedstorage/container/slot/filter/DisabledFluidFilterSlot.java
package com.refinedmods.refinedstorage.container.slot.filter;

import com.refinedmods.refinedstorage.inventory.fluid.FluidInventory;
import net.minecraft.world.item.ItemStack;

import javax.annotation.Nonnull;

public class DisabledFluidFilterSlot extends FluidFilterSlot {
    public DisabledFluidFilterSlot(FluidInventory inventory, int inventoryIndex, int x, int y, int flags) {
        super(inventory, inventoryIndex, x, y, flags);
    }

    public DisabledFluidFilterSlot(FluidInventory inventory, int inventoryIndex, int x, int y) {
        super(inventory, inventoryIndex, x, y);
    }

    @Override
    public void onContainerClicked(@Nonnull ItemStack stack) {
        // NO OP
    }
}

File: src/main/kotlin/com/refinedmods/refinedstorage/container/slot/filter/DisabledFluidFilterSlot.kt
package com.refinedmods.refinedstorage.container.slot.filter

import com.refinedmods.refinedstorage.api.network.grid.IGrid.slotId
import com.refinedmods.refinedstorage.inventory.fluid.FluidInventory
import net.minecraft.item.ItemStack

class DisabledFluidFilterSlot : FluidFilterSlot {
    constructor(inventory: FluidInventory, inventoryIndex: Int, x: Int, y: Int, flags: Int) : super(inventory, inventoryIndex, x, y, flags) {}
    constructor(inventory: FluidInventory, inventoryIndex: Int, x: Int, y: Int) : super(inventory, inventoryIndex, x, y) {}

    override fun onContainerClicked(@Nonnull stack: ItemStack?) {
        // NO OP
    }
}